rotkehlchen/tests/unit/test_inquirer.py

```diff
@@ -138,3 +146,101 @@ def mock_query_price(from_asset, to_asset):
         price = inquirer.find_usd_price(Asset('ETH'), ignore_cache=True)
         assert cc.call_count == 3
         assert price == Price(FVal('2'))
+
+
+def test_all_common_methods_implemented():
+    """Test all current price oracles implement the expected methods.
+    """
+    for oracle in DEFAULT_CURRENT_PRICE_ORACLES_ORDER:
+        if oracle == CurrentPriceOracle.COINGECKO:
+            instance = Coingecko
+        elif oracle == CurrentPriceOracle.CRYPTOCOMPARE:
+            instance = Cryptocompare
+        else:
+            raise AssertionError(
+                f'Unexpected current price oracle: {oracle}. Update this test',
+            )
+
+        # Check 'rate_limited_in_last' method exists
+        assert hasattr(instance, 'rate_limited_in_last')
+        assert callable(instance.rate_limited_in_last)
+        # Check 'query_historical_price' method exists
+        assert hasattr(instance, 'query_current_price')
+        assert callable(instance.query_current_price)
+
+
+def test_set_oracles_default_order(inquirer):
```

Comment:
This is not testing the actual code but just the fixture. This is  a useless test.